### DISCUSSION TITLE
What about emscripten to generate WebAssembly?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#865] [Q&A] [answered]
https://emscripten.org/
Emscripten is a complete compiler toolchain to WebAssembly, using LLVM, with a special focus on speed, size, and the Web platform.

**Comment by paulocoutinhox** [comment]
I have some projects that use it with SDL2:
- https://github.com/nativium/nativium-game
- https://github.com/nativium/nativium-modules
- https://github.com/sqlitedb/sqlitedb

Links for demo are on readme.

**Comment by aismann** [comment]
@paulocoutinhox 
maybe you can start a PR?

**Comment by paulocoutinhox** [comment]
Hi,

You can test first to check if everything is compatible.

Follow this steps and call `emcmake cmake` in the same way that you call cmake to generate project.

https://gist.github.com/WesThorburn/00c47b267a0e8c8431e06b14997778e4

**Comment by aismann** [comment]
@halx99 
What are you mean about support emscripten/WebAssemply?
Maybe am reading a sentence like this ;)
"I'm working on it"

**Reply by halx99** [reply]
@aismann If you make it works, you can share your fork at here.

**Comment by antn9x** [comment]
Hi there, can axmol work with WebAssembly for now?

**Reply by halx99** [reply, marked as answer]
Hi there, but yes, since axmol-2.0 can work with WebAssembly
